util/logtools/generate_triggers.ts

```diff
@@ -1288,6 +1292,60 @@ CastInfo Hints: ${[...castTypeFullSuggestions].join(', ')}
           mapInfo.fights[0]?.instances[0]?.groups?.source ?? 'MISSING SOURCE'
         }', capture: false },
       response: Responses.tankBuster(),
+    },`;
+        break;
+      case 'Stack':
+        triggersText += `
+    {
+      id: '${args.trigger_id_prefix ?? ''} ${abilityName}',
+      type: 'StartsUsing',
+      netRegex: { id: ${allIdsString}, source: '${
+          mapInfo.fights[0]?.instances[0]?.groups?.source ?? 'MISSING SOURCE'
+        }', capture: true },
+      response: Responses.stackMarkerOn(),
+    },`;
+        break;
+      case 'Party Stacks':
+        triggersText += `
+    {
+      id: '${args.trigger_id_prefix ?? ''} ${abilityName}',
+      type: 'StartsUsing',
+      netRegex: { id: ${allIdsString}, source: '${
+          mapInfo.fights[0]?.instances[0]?.groups?.source ?? 'MISSING SOURCE'
+        }', capture: false },
+      infoText: (_data, _matches, output) => output.stacks!(),
+      outputStrings: {
+        stacks: {
+          en: 'Stacks',
+          de: 'Sammeln',
+          fr: 'Package',
+          cn: '分摊',
+          ko: '쉐어',
```

Comment:
Wondering if perhaps this might be better as `Outputs.healerGroups`?  I think that's typically the string we use for paired light-party stacks.  If you prefer the 'Stacks' nomenclature though, perhaps this could be added to `outputs.ts`, just so it's one less place to maintain a translation object?

Reply:
This was copy/pasted from another triggers file. I agree that it should exist as an option in `outputs`, but I didn't want to start modifying other triggers as part of this PR, I'd rather normalize it as a separate PR.

As for `Stacks` vs `Healer Groups`, sometimes the stack markers aren't on healers. I added this as an option when I ran into the `Heartsick` stack markers in r2n, which are on two DPS players, for example.

It might make more sense for this to be `Light Party Stacks` or something, but assuming r2n's Heartsick carries over into r2s, there'll be adjustments/flex needed for DPS anyways, at which point just calling it `Stacks` still makes sense.

Reply:
> This was copy/pasted from another triggers file. I agree that it should exist as an option in `outputs`, but I didn't want to start modifying other triggers as part of this PR, I'd rather normalize it as a separate PR.

Would it be possible to add it to `outputs.ts` and reference here, and we can normalize the other zone files as a separate PR?  I agree it doesn't make sense to tackle all the places where it's used in this PR, but I think it would be good to avoid adding this as a(nother) place it'll need to be fixed later.

> As for `Stacks` vs `Healer Groups`, sometimes the stack markers aren't on healers. I added this as an option when I ran into the `Heartsick` stack markers in r2n, which are on two DPS players, for example.
>
> It might make more sense for this to be `Light Party Stacks` or something, but assuming r2n's Heartsick carries over into r2s, there'll be adjustments/flex needed for DPS anyways, at which point just calling it `Stacks` still makes sense.

The best rule of thumb that I've come up with is that if the mechanic is always resolvable in fixed light parties with no flexing needed, `healerGroups` is the best output since it's a known output we use elsewhere for (functionally) the same thing.  If flexing may be required, iirc how we've handled it in other content, we typically use `stackMarkerOn` or some variant of it to identify which players have the stack, so that at least sufficient info is communicated in the trigger output for people to resolve on their own (since flexing strats may vary).